benchmarks/showcases/LazyGridImageView/nativeAndroidApp/app/src/main/java/org/jetbrains/lazygridimage/MainActivity.kt

```diff
@@ -0,0 +1,162 @@
+package org.jetbrains.lazygridimage
+
+import android.graphics.BitmapFactory
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
```

Comment:
Should it be Compose for "Android Native"? Isn't it supposed to be xml views?

Reply:
>Isn't it supposed to be xml views?

No, it is not. [See](https://docs.google.com/document/d/1GfyXCnKPX4eozPZ2IadArSZECgs6egKyp069Hm7wEHc/edit?tab=t.0)